zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -133,7 +133,7 @@ public static Either<ProblemDetail, DecisionInstanceQuery> toDecisionInstanceQue
     return buildSearchQuery(filter, sort, page, SearchQueryBuilders::decisionInstanceSearchQuery);
   }
 
-  private static DecisionInstanceFilter toDecisionInstanceFilter(
+  private static io.camunda.service.search.filter.DecisionInstanceFilter toDecisionInstanceFilter(
```

Comment:
❓ Do we need these explicit imports? Do we use the same class from the REST package as well?

Reply:
No, we use the number key variants which are called differently.
It's only needed due to the `*` imports. I'll fix that.